File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.cs
public class Solution {
    public int MaximumCount(int[] nums) {
        int p = 0 , n = 0;
        for(int i = 0 ; i < nums.Length; i++){
            if(nums[i] < 0)n++;

            if(nums[i] >0){
                p = nums.Length - i;
                break;
            }
        }
        return Math.Max(p,n);
    }
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/2614-maximum-count-of-positive-integer-and-negative-integer.cpp
class Solution {
public:
    int maximumCount(vector<int>& nums) {
        int n = lower_bound(nums.begin(),nums.end(),0)-nums.begin();
        int p = nums.size() - (upper_bound(nums.begin(),nums.end(),0)-nums.begin());
        return max(p,n);
    }
};